tests/sles4sap/wmp_basic_test.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run Workload Memory Protection basic test
+# Maintainer: Alvaro Carvajal <acarvajal@suse.de>
+
+use base "sles4sap";
+use testapi;
+use utils qw(zypper_call file_content_replace);
+use version_utils qw(is_sle);
+use strict;
+use warnings;
+
+sub run {
+    my ($self)     = @_;
+    my $testname   = 'wmp_basic_tests';
+    my $logdir     = '/root/wmp_logs';
+    my $sapsvc     = '/usr/sap/sapservices';
+    my @testphases = qw(initial takeover takeback);
+
+    # WMP is a feature of SLES for SAP Applications 15+. Skip test in older systems
```

Comment:
Is this test really needed? As it is already done in `wmp_setup.pm`?

Reply:
Python script will fail without the WMP setup, so it's either skip the test when not in SLES4SAP 15+, or leave it fail. I think it makes more sense having the check in both modules, but perhaps the verification in this module can be change so it doesn't check for SLES4SAP 15+ and instead checks for WMP having been configured. Will see how to do that.

Reply:
Ok, do what is easier :-)

Reply:
Done. Now test module will check for `sap.slice` cgroup to be configured (this is done in `tests/sles4sap/wmp_setup`) before proceeding instead of checking for the OS version.

Reply:
Verification run for this change: http://mango.suse.de/tests/2703#step/wmp_basic_test/3

We could also make the test module fail, but IMO it's better this way to avoid tests runs failing due to test misconfiguration.